src/idiomatic/leveraging-the-type-system/extension-traits/trait-method-conflicts.md

```diff
@@ -0,0 +1,62 @@
+---
+minutes: 5
+---
+
+# Trait Method Conflicts
+
+What happens when you have a name conflict between two different trait methods
```

Comment:
We also need a setup here. This one is simple: imagine that Ext1 and Ext2 come from two different crates, and they chose the name "is_palindrome" independently, without coordination.